plugins/auto_merge/checks/is_title_using_conventional_commit.go

```diff
@@ -0,0 +1,30 @@
+package checks
+
+import (
+	"regexp"
+
+	"github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/config"
+	"github.com/xanzy/go-gitlab"
+)
+
+var conventionalCommitSpecUrl = "https://www.conventionalcommits.org/en/v1.0.0/#specification"
+var ConventionalCommitRegex = regexp.MustCompile(`^(feat|fix|docs|style|refactor|perf|test|chore)(?:\((.*)\))?(\!)?\: (.*)$`)
+
+type IsTitleUsingConventionalCommit struct {
+}
```

Comment:
Why is this struct empty?

Reply:
@lukashass The "class" has no internal variables.